core/storage/compacter/compacter.cpp

```diff
@@ -234,6 +234,9 @@ namespace fc::storage::compacter {
       }
       ts = ts_load->load(ts->getParents()).value();
     }
+    auto head_res{interpreter_cache->get(head->key).value()};
```

Comment:
Could it has an error?

Reply:
No, chain head must be interpreted and cached.

Reply:
ok

Reply:
No, program keeps `head` consistent/correct, so there should not be error